acme/acme.go

```diff
@@ -549,16 +593,36 @@ func (a *ACME) LoadCertificateForDomains(domains []string) {
 		} else {
 			domain = Domain{Main: domains[0]}
 		}
+
 		if _, exists := account.DomainsCertificate.exists(domain); exists {
-			// domain already exists
+			log.Debugf("ACME Certificate for domain %v exists.", domain)
 			return
 		}
+
+		// Check if our domains are matching our ingoreFilters
+		if len(a.IgnoreFilters) > 0 {
```

Comment:
In case we are in a 'onDemand' for now there is no check with `ignoreFilters`, this can be added if needed.